### PR TITLE
Issue 1133 lon labels freaks out

### PR DESCRIPTION
**Pull request by doutriaux1**

### CODE REVIEW

**Comment by doutriaux1** [comment]
goes with https://github.com/UV-CDAT/uvcdat-testdata/pull/29

**Comment by doutriaux1** [comment]
@chaosphere2112 @williams13 @aashish24 @dlonie ready to be merged

File: testing/vcs/test_lon_axes_freak_out.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import os,sys,vcs,cdms2
+src=sys.argv[1]
+pth = os.path.join(os.path.dirname(__file__),"..")
+sys.path.append(pth)
+import checkimage
+
+
+f=cdms2.open(os.path.join(sys.prefix,"sample_data","clt.nc"))
```
Comment (dlonie) [inline testing/vcs/test_lon_axes_freak_out.py]: Use vcs.prefix instead of sys.prefix to ensure that this will work with system python builds.

**Comment by chaosphere2112** [comment]
:+1: Looks good to me, fixes the issue we were having earlier. Sadly, does not fix my issue we found this morning, which appeared to be related :cry:.

File: testing/vcs/test_lon_axes_freak_out.py (hunk 1 above)
Comment (doutriaux1) [inline testing/vcs/test_lon_axes_freak_out.py]: @dlonie good idea, we should create an issue to update all tests I'm sure it's everywhere.

**Comment by chaosphere2112** [comment]
Yup, this totally fixes the axis issue that @williams13 noticed; I'm going to merge!

**Pull request merged**